website/components/mdx/_cloudquery-sign-in.mdx

```diff
@@ -8,9 +26,8 @@ You will be able to create a new account when you sign in with the CLI for the f
 
 To sign in from the CLI, run the following command:
 
-```bash
-cloudquery login
-```
+<SignIn os={props.os} />
```

Comment:
This is the fix, pass the prop from each individual OS guide to the `SignIn` to render a different command